src/dialogs/more-info/controls/more-info-media_player.js

```diff
@@ -97,6 +97,19 @@ class MoreInfoMediaPlayer extends LocalizeMixin(EventsMixin(PolymerElement)) {
         </paper-listbox>
       </paper-dropdown-menu>
     </div>
+    <!-- SOUND MODE PICKER -->
+    <div class="controls layout horizontal justified">
+      <template is='dom-if' if='[[!computeHideSelectSoundMode(playerObj)]]'>
+        <iron-icon class="source-input" icon="mdi:music-note"></iron-icon>
+        <paper-dropdown-menu class="flex source-input" dynamic-align label-float label='Sound Mode'>
+          <paper-listbox slot="dropdown-content" selected="{{soundModeIndex}}">
```

Comment:
You can use `attr-for-selected` so you don't have to use the index

Reply:
I am not sure how 'attr-for-selected' would work. I just made the method identical to the 'source input dropdown menu' that is already in the file. I tested how it works now, and it is working fine.
I suggest keeping the 'soundModeIndex' as it is now, and someone that knows how 'atrr-for-selected' works can update this later and at the same time change the source input that now also uses 'sourceIndex'

Reply:
it's documented here: https://www.webcomponents.org/element/polymerelements/paper-listbox/demo/demo/index.html

Reply:
Just implemented "atrr-for-selected".
Thanks for your help @c727